src/components/app/details/triggerView/ciMaterial.tsx

```diff
@@ -39,6 +39,14 @@ export class CIMaterial extends Component<CIMaterialProps, CIMaterialState> {
 
     componentDidMount() {
         this.getSecurityModuleStatus()
+        let envId = this.state.selectedCIPipeline?.environmentId
```

Comment:
We can remove all these logic and directly assign the default in selected env

Reply:
Here have been slight cases, if we have env configured in build stage, we need to show that in trigger stage, but if someone chooses different env in trigger stage , we need to show that in trigger , and that also has two cases for default-ci and other environments.